src/js/tech/html5.js

```diff
@@ -1010,7 +1010,7 @@ Html5.canOverrideAttributes = function() {
  *         - False otherwise
  */
 Html5.supportsNativeTextTracks = function() {
-  return browser.IS_ANY_SAFARI;
+  return browser.IS_ANY_SAFARI || (browser.IS_IOS && browser.IS_CHROME);
```

Comment:
does chrome for iOS handle text tracks like iOS safari does? Are there caption settings available at the browser level or does it use the OS-level settings?

Reply:
Right now, Chrome and Safari on iOS handles text tracks in the same way  (Chrome, same as Safari, uses OS level settings for captions and subtitles).

Reply:
cool wanted to confirm that's how it goes.